.github/workflows/wasmcloud.yml

```diff
@@ -7,16 +7,38 @@ on:
     branches:
     - main
     tags:
+    - 'actor-v[0-9].[0-9]+.[0-9]+'
```

Comment:
Since there's a long list here now, what if we narrowed down to host and crates generally:

crates:
```
*-v[0-9].[0-9]+.[0-9]+
*-v[0-9].[0-9]+.[0-9]+-*
```

host:
```
v[0-9].[0-9]+.[0-9]+
v[0-9].[0-9]+.[0-9]+-*
```

Do we get anything out of enumerating?

Reply:
I just like to be explicit to ensure we're not accidentally hitting some case we did not consider, but this is probably fine - I'll apply this change

Reply:
No, I will not be changing this, because this workflow should not run on e.g. chart tags or UI tags:
- https://github.com/wasmCloud/wasmCloud/blob/7c8ee2af3ce17b5515c673358d82634639286796/.github/workflows/wasmcloud-chart.yml#L9
- https://github.com/wasmCloud/wasmCloud/blob/7c8ee2af3ce17b5515c673358d82634639286796/.github/workflows/washboard.yml#L10